fiftyone/core/map/threading.py

```diff
@@ -208,8 +212,15 @@ def get_results(
 
             # This is for the global progress bar.
             if progress is True:
-                results = tqdm(
-                    results, total=sum(batch.total for batch in sample_batches)
-                )
-
-            yield from results
+                with fou.ProgressBar(
+                    total=sum(batch.total for batch in sample_batches),
+                    progress=progress,
+                ) as pb:
+                    # Wait for all workers to finish
```

Comment:
I thought we are yielding as a worker finishes computation, so we do not need to wait for all workers to finish.

Reply:
this is an errant comment, sorry for the confusion. This will not be blocking until all workers finish